internal/commands/doc.go

```diff
@@ -14,8 +15,12 @@ func NewCmdDoc() *cobra.Command {
 		Long:                  "Create docs with ize commands description",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
+			err := os.MkdirAll("./docs", 0777)
+			if err != nil {
+				return err
+			}
 
-			err := doc.GenMarkdownTree(cmd.Root(), "./commands")
+			err = doc.GenMarkdownTree(cmd.Root(), "./docs")
```

Comment:
Let's update to the relevant directory where it will be picked up by the website.